suite-native/module-home/src/components/PortfolioGraph.tsx

```diff
@@ -37,16 +39,22 @@ export const PortfolioGraph = () => {
     return (
         <VStack spacing="large">
             <PortfolioGraphHeader />
-
-            <Graph
-                points={graphPoints}
-                loading={isLoading}
-                onPointSelected={setSelectedPoint}
-                onGestureEnd={setInitialSelectedPoints}
-                onTryAgain={refetch}
-                error={error}
-            />
-            <TimeSwitch selectedTimeFrame={timeframe} onSelectTimeFrame={onSelectTimeFrame} />
+            {!isDiscoveryActive && (
```

Comment:
I don't think we should hide graph during discovery just show it's loading state.

Also I am not sure if this is enough to prevent refetch of graph because I think most of the magic is happening in hook `useGraphForAllAccounts`

Reply:
Graph behaviour changed show loader and do not fetch data while discovery in 019dcf358.